lib/digitizer_block_impl.cc

```diff
@@ -989,7 +988,7 @@ namespace gr {
          }
 
          // Relax CPU
-         boost::this_thread::sleep_for(boost::chrono::microseconds(100));
+         std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
two thing:
* (existed before): hard-coding of wait/idle/yield duration -> not so nice. Should be derived from actual polling duty cycle
* changed the sleep from `microseconds` to `milliseconds` ... guess this was by mistake.

Reply:
For the hard coded wait duration:  I agree, though thats a bit out of scope of this MR.

For `milliseconds` --> `microseconds`: gosh, that struck me hard, its from debugging :F . In the future I will tripple check what I commit/pushed. Fixed.